src/chrome_debugger.rs

```diff
@@ -33,9 +34,25 @@ impl ChromeRemoteDebugger {
             ws_url: String
         }
 
-        let targets: Vec<Target> = reqwest::blocking::get(
-            format!("http://localhost:{port}/json/list")
-        )?.json()?;
+        // create targets (if error, retry in 1 second) (max 10 tries)
+        let mut targets: Vec<Target> = Vec::new();
+        let mut retry = 0;
+        while targets.len() == 0 {
+            if retry > 10 {
+                panic!("Failed to get targets");
+            }
+            match reqwest::blocking::get(
+                format!("http://localhost:{port}/json/list", port = port)
+            ) {
+                Ok(res) => {
+                    targets = res.json()?;
+                },
+                Err(_) => {
+                    retry += 1;
+                    thread::sleep(time::Duration::from_secs(1));
+                }
+            }
+        }
 
         let selected = targets.first().context("no debugging targets found")?;
```

Comment:
it's safe to use unwrap here since we know the targets Vec won't be empty
```suggestion
        let selected = targets.first().unwrap();
```